js_course/test-2.spec.js

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
+
+test('test-2', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
```

Comment:
Please, check this doc - https://playwright.dev/docs/api/class-testoptions#test-options-base-url
You can add `https://www.saucedemo.com` in `playwright.config.js` file.
Then in test you will just write 

```
await page.goto('/');
```

Please, change here - https://github.com/divchunonka/js_course/pull/1/files#diff-8421b2f4ad242eb670ea84ae0057d8708f0370adad5ac063da20b8abc54426daR28